### ISSUE TITLE
Can't change / set destination account in rule triggers

### ISSUE THREAD

**Issue by leppa** [firefly-iii/firefly-iii#3853]
**Bug description**
I am running Firefly III version 5.4.2, and my problem is:

When I edit value of "Destination account name is..." in rule triggers I don't get any autocompletion. If I press *See matching transactions*, I'm presented with the correct list of transactions (so the rule is correct). However, when I press *Update rule* it produces the following error: "This value is invalid for the selected trigger."

Same issue if I change "Destination account name is..." to "Destination account name contains...". Didn't try with other triggers.

**Steps to reproduce**
1. Edit existing rule.
2. Try changing destination account name trigger to any valid expense account.

**Extra info**

Screenshots:

![Opera Snapshot_2020-09-24_173747](https://user-images.githubusercontent.com/857627/94171266-5a7bd780-fe91-11ea-9115-08d3377d1d6f.png)

![Opera Snapshot_2020-09-24_180835](https://user-images.githubusercontent.com/857627/94171302-64053f80-fe91-11ea-830a-1986023c17ba.png)

Docker image on Raspberry Pi. Debug information generated at 2020-09-24 18:01:07 Europe/Berlin for Firefly III version **5.4.2**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.4.2 |
| Firefly III API | 1.4.0 |
| PHP | 7.4.10 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | d73d6f6a-e7e8-4a5c-99cf-33ee4649147c |
| Using docker? | true |
| Telemetry | true |
| Layout | v1 |
| App environment | local |
| App debug mode | false |
| App cache driver | memcached |
| App logging | warning, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | de_DE |
| BCscale | 12 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| DB version | 15 (exp. 15) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2020-09-01 00:00:00 |
| Session end | 2020-09-30 23:59:59 |
| Session first | 2020-07-27 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 Safari/537.36 OPR/71.0.3770.171 |

**Bonus points**

- I searched and nobody reported this bug before.
- I have added a screenshot.
- I was able to replicate it on the demo site https://demo.firefly-iii.org/

**Comment by sephrat** [comment]
Yes autocomplete seems to be broken for source and destination accounts.

`create-edit.js?v=5.4.2:369 Select list value is source_account_starts, destroy auto complete, do nothing else.`
Code in create-edit expected 'from_account_starts' instead.
Similar logic with destination and to.

Seems to be an easy fix, I'll try to submit a PR tomorrow.

**Comment by JC5** [comment]
:+1:.

Will be part of the next release. Thanks, @sephrat.

**Comment by marcelbrueckner** [comment]
This seems not to be fixed. I've encountered the same problem on my Firefly III installation on v5.4.4. I was able to reproduce it on the demo site.

![grafik](https://user-images.githubusercontent.com/3597890/97121176-961df180-171c-11eb-85e1-bd681906926b.png)
![grafik](https://user-images.githubusercontent.com/3597890/97121177-9918e200-171c-11eb-9668-76bf287a84bb.png)
![grafik](https://user-images.githubusercontent.com/3597890/97121180-9b7b3c00-171c-11eb-87bc-36ca708bc431.png)
